solutions/standard/stack_definition.json

```diff
@@ -701,6 +701,10 @@
           "name": "add_container_name_suffix",
           "value": true
         },
+        {
+          "name": "cc_pipeline_properties",
+          "value": "[{\"app-name\": \"rag-sample-app\"}]"
```

Comment:
I think it should follow the schema like
```json
[
  {
    "pipeline_id": "cc",
    "properties": [
      {
        "name": "app-name",
        "type": "text",
        "value": "...."
      }
...
```

Reply:
You're right - will make the change

Reply:
It should follow the right schema now, I wrongly assumed it was using the same approach as DA-defined variables.